src/views/PostView.vue

```diff
@@ -53,8 +53,10 @@ const loadMarkdown = () => {
   const module = modules.value.find(item => item.meta.title === title && item.meta.seriesTitle === sTitle)
 
   peekData.value = module.meta
-  content.value = DOMPurify.sanitize(getContentHtml(module.raw))
+  const htmlContent = getContentHtml(module.raw)
+  content.value = DOMPurify.sanitize(htmlContent[0])
   toc.value = getTableOfContent(module.raw)
+  peekData.value.description = htmlContent[1].text
```

Comment:
destructure the value. Use the variable names that fit best
```suggestion
  const [html, description] = getContentHtml(module.raw)
  content.value = DOMPurify.sanitize(html)
  toc.value = getTableOfContent(module.raw)
  peekData.value.description = description.text
```